src/main/java/au/org/emii/ncdfgenerator/NcdfDefinitionXMLParser.java

```diff
@@ -1,276 +1,374 @@
 
 package au.org.emii.ncdfgenerator;
```

Comment:
Agree. New files/functionality are being dumped in the top level dir/package without a lot of thought. I'd prefer to wait to address this, when there's more time to consider the best way to organize the sources.